samples/Azure.AI.FormRecognizer/Generated/AzureAIFormRecognizerModelFactory.cs

```diff
@@ -14,7 +14,7 @@ namespace Azure.AI.FormRecognizer.Models
     /// <summary> Model factory for read-only models. </summary>
     public static partial class AzureAIFormRecognizerModelFactory
     {
-        /// <summary> Initializes a new instance of ErrorInformation. </summary>
+        /// <summary> Initializes a new instance of <see cref="ErrorInformation"/>. </summary>
         /// <param name="code"></param>
         /// <param name="message"></param>
         /// <returns> A new <see cref="FormRecognizer.Models.ErrorInformation"/> instance for mocking. </returns>
```

Comment:
Why these two ErrorInformation are different?

Reply:
Both works.
Line 17 is the result of writer.Append(string), Line 20 is the result of writer.Append(CSharpType)

Reply:
I mean would it be better to keep them consistent?

Reply:
Thank you for pointing this out - initially I did not see this issue, but this really uncovers an issue during the regeneration.
Please see the CI failures here: https://github.com/Azure/azure-sdk-for-net/pull/29039/checks?check_run_id=6668741619
Fully qualified name here is better because we really could have some cases that the cref here with only a name can be ambiguous 
I will push an update for this soon

Reply:
@pshao25 these two are now aligned with each other, and the issue above is resolved.